docs/userGuide/gettingStarted.md

```diff
@@ -13,6 +13,7 @@
   %%{{ icon_ticked }}%% a basic knowledge of [Markdown](https://www.markdownguide.org/basic-syntax/) and [HTML](https://www.w3schools.com/html/) syntax<br>
   %%{{ icon_ticked }}%% a basic knowledge of running CLI commands<br>
   %%{{ icon_ticked }}%% a recent version of [npm](https://www.npmjs.com/get-npm) installed
+  %%{{ icon_ticked }}%% [Node v8.11.3](https://nodejs.org) or higher installed
```

Comment:
- `[Node v8.11.3](https://nodejs.org)` → `[Node.js](https://nodejs.org) v8.11.3`

- Node.js v8 is reaching End-of-life on 2019-12-31: https://nodejs.org/en/about/releases/
  That's less than 3 weeks from now.
  Let's recommend v10.13.0 (LTS) or higher; v10 will only reach End-of-life on 2021-04-30.

Reply:
👍. Fixed thanks!

Reply:
@hcwong the version number should be outside of the link, which is what the first bullet point by @acjh meant.

Reply:
Oops, sorry got it

Reply:
Also, `Node` → `Node.js`.